cmd/docker/docker.go

```diff
@@ -337,6 +337,9 @@ func findCommand(cmd *cobra.Command, commands []string) bool {
 }
 
 func isSupported(cmd *cobra.Command, details versionDetails) error {
+	if isNoRemote(cmd) {
+		return nil
```

Comment:
What about commands which are "no-remote" but also "experimentalCLI", as this is an annotation only client side and so does not need the remote informations ?

Reply:
Good question, indeed